server/ormconfig.ts

```diff
@@ -42,10 +44,12 @@ export const ormConfig: PostgresConnectionOptions = {
     UserNotificationSetting,
     OrphanedRecord,
     OrphanedRecordAction,
+    Observation,
+    ObservationType,
   ],
   namingStrategy: new SnakeNamingStrategy(),
   synchronize: (env.isDev && process.env.SYNC_DATABASE === 'true'),
-  logging: false,
+  logging: (env.isDev || process.env.ORM_LOGGING === 'ture'),
```

Comment:
Remove. Or adjust to 1) fix the type in `true` and 2) maybe change `(env.isDev || ...` to `(env.isDev && ...` - I personally don't want the logging on just because I'm in development mode.

Reply:
Fixed